static/scripts/login.js

```diff
@@ -137,6 +138,17 @@ $(document).ready(function() {
         $school.trigger('change');
     }
 
+    // EBS-System | Alert
+    $.ajax({
+        url: '/alerts',
+        contentType: 'application/json',
+        dataType: 'json',
+        success(result) {
+            result.forEach((message) => {
+                $('.alert-section').append(loginMessageBuilder(message));
```

Comment:
there before, clear all existing messages from within the alert section otherwise on soft page changes the list will be expanded again and again...